src/engine/renderer/tr_model_iqm.cpp

```diff
@@ -631,9 +631,14 @@ bool R_LoadIQModel( model_t *mod, void *buffer, int filesize,
 	for(unsigned i = 0; i < header->num_frames; i++ ) {
 		pose = ( iqmPose_t* )IQMPtr( header, header->ofs_poses );
 		for(unsigned j = 0; j < header->num_poses; j++, pose++, trans++ ) {
-			vec3_t	translate;
+#ifdef __SANITIZE_ADDRESS__
```

Comment:
The root cause is doing 16-byte reads with _mm_loadu_ps on 12-byte vec3_t's. It would be better to fix that at the source, something like https://github.com/slipher/Daemon/commit/53af9196861876bf664ba820905272dbe2e3debf